services/wercker/wercker.service.js

```diff
@@ -14,6 +14,24 @@ const werckerSchema = Joi.array()
   .max(1)
   .required()
 
+const werckerCIDocumentation = `
+<p>
+  Note that Wercker badge Key (used in Wercker's native badge urls) is not the same as
+  the Application Id and the badge key will not work.
+</p>
+<p>
+  You can use the Wercker API to locate your Application Id:
```

Comment:
I've no idea if there's a UI/more user friendly means to look this up, so pointing folks to the API since we know that'll work

Reply:
Hmmm... Given that it's the main ID used for quite a few of the API calls and given that users haven't complained about this, I would be surprised if it wasn't available in the UI. But no way to verify that without a real account, so let's go with that for now. 😃